### PR TITLE
# Update disabled states tokens

### PR DESCRIPTION
**Pull request by 3dyonic**
https://monday.monday.com/boards/245345663/pulses/2618132698
- Add opacity-40 with 0.4 value to opacity registration tokens.
- Add "disabled-component-opacity" token.
- Add Basic token: --color-asphalt-muted
- Add Basic token:--color-ui_grey-muted
- Change values for:
 --disabled-background-color
 --disabled-background-on-secondary-color
 --disabled-text-color
 --disabled-text-on-secondary-color
 - Hard code changes in dark , black and hacker theme. (per current convention)

### CODE REVIEW

File: src/core/_core-maps.scss [hunk 1]
```diff
@@ -9,3 +9,5 @@
 @import "maps/core-shadows-map";
 @import "maps/core-spacing-map";
 @import "maps/core-typography-map";
+//
```
Comment (laviomri) [inline src/core/_core-maps.scss:12]: is this `//` intentional?

**Comment by laviomri** [comment]
It looks good to me (especially the stylelint rule 👍 ), but I think I'm missing a bit of context to approve it..

File: src/core/_core-maps.scss (hunk 1 above)
Comment (3dyonic) [reply src/core/_core-maps.scss:12]: yes, it's an exception, common scope is a small anti pattern, just wanted a small separation. 
its a common thing in SCSS loaders to separate using a line break or // 
i can add a line break if this is more clear

Comment (3dyonic) [reply src/core/_core-maps.scss:12]: > It looks good to me (especially the stylelint rule 👍 ), but I think I'm missing a bit of context to approve it..

The Task is attached.
The disabled colors (text,background) where defined with Opacity, now we corrected this. 
added new color tokens and reassigned the disabled tokens with the new values.
currently still in the old method. soon to be replaced.

just a routine token update.

Comment (laviomri) [reply src/core/_core-maps.scss:12]: > yes, it's an exception, common scope is a small anti pattern, just wanted a small separation. its a common thing in SCSS loaders to separate using a line break or // i can add a line break if this is more clear

I think I understand. Maybe it's just me, but it was a bit confusing. A line break would have been more intuitive for me :) but that's up to you

**Review by laviomri — APPROVED** [review]

File: src/core/_core-maps.scss (hunk 1 above)
Comment (3dyonic) [reply src/core/_core-maps.scss:12]: line breaks can be accidental. leaving it for the time being.
when we will have some luxury time will can define SCSS commenting and related standards.

**Review by orrgottlieb — APPROVED** [review]

**Pull request merged**